### PR TITLE
Mentions

### PR DESCRIPTION
**Pull request by SaFrMo**
These should be the only changes to libfritter required for mentions! I figured I could submit the PR for this now and do the PR for the front-end when it's ready in a day or two, just to avoid throwing them all at you at once. Let me know if you'd like to see any changes!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -159,7 +165,13 @@ function setHooks (inst) {
         await db.notifications.delete(url)
       }
     })
+
+    // mention notifications
+    db.posts.on('put-record', async ({record, url, origin}) => {
+      //if (origin === inst.userUrl) return // dont index the user's self-mentions
```
Comment (pfrazee) [inline index.js]: We need this, yeah?

**Comment by pfrazee** [comment]
Looks great. I have one comment. Could you also add some test cases?

File: index.js (hunk 1 above)
Comment (SaFrMo) [reply index.js]: Hah, yep! Added back in.

**Comment by SaFrMo** [comment]
Sure thing! Added in the latest commit.

**Comment by pfrazee** [comment]
Excellent work @SaFrMo! Published `2.2.0`

**Pull request merged**